test/visual_regression/tests/discover/chart_visualization.js

```diff
@@ -48,32 +48,38 @@ export default function ({ getService, getPageObjects }) {
     });
 
     describe('query', function () {
-      this.tags(['skipFirefox']);
```

Comment:
I assume this was here for a reason, @dmlemeshko maybe it would be best to leave comments or links to issues for `skipX` tags to explain why a suite was skipped.

Reply:
This tag was a carryover from the functional tests. The visual regression tests are still run in both chrome and firefox which is why I removed this.

Are you wanting me to put it back or add a different tag?

Reply:
The visual regression tests are a little complicated, they generate snapshots from Chrome only, and then are shipped to the Percy service where the snapshot is rendered into screenshots in both Chrome and Firefox. This tag only influences whether tests are run in the `firefox smoke` ci group. If you didn't mean to add this suite to those tests I'd suggest putting it back.